segmentstore/server/src/test/java/io/pravega/segmentstore/server/containers/StreamSegmentContainerTests.java

```diff
@@ -2110,6 +2119,164 @@ public void testSnapshotInfo() throws Exception {
         container.stopAsync().awaitTerminated();
     }
 
+
+    /**
+     * Test that the {@link ContainerEventProcessor} service is started as part of the {@link StreamSegmentContainer}
+     * and that it can process events.
+     *
+     * @throws Exception
+     */
+    @Test(timeout = 10000)
+    public void testBasicContainerEventProcessor() throws Exception {
```

Comment:
Since these unit tests are inside the Segment Container unit tests, let's group them by adding a common prefix. For example `testEventProcessor***` so when someone runs them, they at least get a sense of what they're testing here.

I see some of your tests already have "event processor" somewhere in their name - that's fine - this comment applies to the others.

Reply:
Done.